src/components/shared/InlineMessage.js

```diff
@@ -33,7 +33,7 @@ export const InlineMessage = ({
           <Icon color={color} name={SVG} size={iconSize} />
         </Box>
       )}
-      <Paragraph color={color || fontColor} size={fontSize}>
+      <Paragraph color={color !== 'info' ? color : fontColor} size={fontSize}>
```

Comment:
If I were to set the `fontColor` but not `color`, then there would be no effect. I think what we should do here is just set the default `fontColor` to info an explicitly set it when we want it to change.

Reply:
Correction: if we set color and fontColor, fontColor would have no effect

Reply:
I've defined the `types` to specify default settings for each `type` and adjusted the codebase and is now ready for another look. Good call! Thank you @davidsmejia 👍